packages/app/src/embed/components/Actions/index.js

```diff
@@ -27,7 +26,7 @@ export function GlobalActions({
   previewVisible,
   initialPath,
 }) {
-  const experimentPromise = useExperimentResult('embed-open-wording');
+  const experimentPromise = useExperimentResult('embed-open-wording-iteration2');
```

Comment:
Just make sure you created this experiment in both environments: production and staging.

Reply:
Yeah, sure thing!